### PR TITLE
Add StereoConvergence settings to various gameini files

### PR DESCRIPTION
**Pull request by SMarioMan**
Naming changed for the StereoConvergence setting, and my old PR was having rebase issues, so I've made a new PR to account for this.

### CODE REVIEW

**Comment by SMarioMan** [comment]
@Armada651 These values should all be ready to go with the new slider implementation.

**Comment by Armada651** [comment]
@JosJuice I'm not really familiar with the GameINI structure, do these changes look good to you?

File: Data/Sys/GameSettings/NAE.ini [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# NAEE01 - Paper Mario
+# NAEP01, NAEE01 - Paper Mario
```
Comment (JosJuice) [inline Data/Sys/GameSettings/NAE.ini]: It's not too important, but these IDs should preferably be sorted alphabetically.

File: Data/Sys/GameSettings/R92.ini [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# R92E01, R92P01, R92J01 - Pikmin 2 Wii
```
Comment (JosJuice) [inline Data/Sys/GameSettings/R92.ini]: Same here. (J should go before P)

**Comment by JosJuice** [comment]
@Armada651 I can't comment on the stereoscopy values, but the rest looks fine other than the nitpicks I mentioned above.

File: Data/Sys/GameSettings/R92.ini (hunk 2 above)
Comment (SMarioMan) [inline Data/Sys/GameSettings/R92.ini]: The non-alphabetical order is based on the order they're listed in the wiki for their associated pages, but I see now that it should be alphabetical like the rest of the inis. Thanks for scrutinizing things! I'll take care of this soon.

**Comment by SMarioMan** [comment]
@JosJuice I corrected the game IDs. 
The stereoscopy values have been tested and work well. They are admittedly subjective (hence the use of a percentage slider) and do not work well in all scenes, which is understandable. They are generally set to be good values for the main parts of the game, as it isn't possible to optimize for more than one convergence at a time.

**Comment by Armada651** [comment]
@JosJuice I trust @SMarioMan on the stereoscopy settings. Besides, any setting at all is better than the default. If there is no existing convergence setting in the GameINI we can just merge it blindly.

File: Data/Sys/GameSettings/GL5.ini [hunk 3]
```diff
@@ -17,3 +17,5 @@ EmulationIssues = Slow
 [ActionReplay]
 # Add action replay cheats here.
 
+[Video_Stereoscopy]
+StereoConvergence = 1
```
Comment (Armada651) [inline Data/Sys/GameSettings/GL5.ini]: @SMarioMan I see multiple games with extremely low convergence settings such as this. Would support for fractions like `0.25` be useful to you here?

**Comment by eckso** [comment]
So all in all, how is the naming settled to StereoConvergence, StereoConvergencePercentage, or StereoConvergenceMinimum? Same with StereoDepth.

I have many games configured both GC, and Wii, with depth. Some of them differ with this PR. Now they were configured back 6 months ago so not sure whether they hold up (my tests yesterday told me they do)
G8M 120, 28
GFZ  6, 47
GLM 500, 32
GM4 151, 60
GMS 283, 30

new ones:
GF7 22, 59
GFZ 6, 47
R2V 31, 42
R4E 8, 35
RMC 442, 21
SF8 10, 14

**Comment by Armada651** [comment]
@eckso All games should get a game-specific `StereoConvergence` setting which you can adjust globally by setting the `StereoConvergencePercentage`. That setting will likely stay in its current incarnation, perhaps with the addition of fractions.

`StereoDepth` is a global setting that should reflect your IPD. However many games have a different scale, so this IPD should be adjusted per-game with the `StereoDepthPercentage` setting. But I'm not accepting GameINI PRs for that setting yet, as it is likely to change soon:

The `StereoDepth` parameter currently describes an offset in pixels, but it will be changed to reflect your IPD in mm. And every game will get a `MeterInUnits` setting that reflects the scale at which the game is running. This new setting will be crucial for VR to get accurate IPDs and positional tracking. Stereoscopy will also benefit from the more accurate IPDs as a result.

But to implement this new IPD setting I will have to implement a method to detect the monitor PPI so I can accurately convert the value in mm to a value in pixels.

File: Data/Sys/GameSettings/GL5.ini (hunk 3 above)
Comment (SMarioMan) [inline Data/Sys/GameSettings/GL5.ini]: Whole integers have worked well on all games I've tested. I don't know how useful it would be.

**Comment by SMarioMan** [comment]
@eckso In my experience, the values you've been using will work just fine if you rename ```StereoConvergenceMinimum``` to ```StereoConvergence``` in your inis. Depth remains unaffected by this PR or any of the other recent adjustments to how 3D is handled in Dolphin. The values I picked are somewhat subjective in nature, so it isn't surprising if some of yours differ.

**Comment by eckso** [comment]
So global is StereoConvergencePercentage and StereoDepth, and per-game is StereoConvergence and StereoDepthPercentage. Isn't that a bit too much of confussion?
On Depth wouldn't any future PR you have in mind (like the IPD in mm) be scalable with current values (as the ones I measured)? Take the last example of 14px, once you find a way to extract inch size from monitor EDID do some maths like below to get PPI, then 14/(86.27/25.4mm<--one inch)=3.4mm

w=1280.0
h=1024.0
diag=19

a = w/(sqrt(pow(diag,2)/(1.0+pow((h/w),2)))) = 86.27PPI

@SMarioMan Global convergence you mean in gfx_opengl.ini? Isn't that setting a proxy so when GameINIs are loaded they copy there?

**Comment by Armada651** [comment]
@eckso That's one of the reasons to change the depth settings. Once we have a setting that describes the game scale we can do away with `StereoDepthPercentage`.

Also, I see that currently the `StereoDepth` setting doesn't accurately describe pixels, it's a pretty arbitrary value right now. I'm currently trying to see what we can do to get the math right on the IPD setting.

**Comment by SMarioMan** [comment]
@eckso When I say "global convergence," I am referring to the old '''StereoConvergenceMinimum'''. That value was not copied from the gameini, but, rather added to the gameini. As such, if you used a value other than 0 in your global config, you'll need to add that to all of your old '''StereoConvergenceMinimum''' values to get a valid new '''StereoConvergence''' value.

**Comment by Armada651** [comment]
@eckso According to some [slides from Nvidia](http://www.nvidia.com/content/gtc-2010/pdfs/2010_gtc2010.pdf) the IPD in monitor pixels just defines an upper limit of the distance between the cameras in pixels.

So I don't need the EDID info, all I have to do is take the camera distance in units calculated with the IPD multiplied with the `MeterInUnits` parameter and then divide it by the screen width to get the correct separation value.

**Pull request merged**